recipes-bsp/grub/grub-xen-pvh_2.04.bb

```diff
@@ -0,0 +1,4 @@
+require grub-xen.inc
+GRUBPLATFORM = "xen_pvh"
+GRUB_TARGET = "i386"
```

Comment:
Why does xen_pvh build as a 32 bit binary?  This is just how xen supports it or was there some other special reason?